node/middlewares/extract.ts

```diff
@@ -19,7 +19,17 @@ interface PersistedQuery extends Query {
 const isPersistedQuery = (query: any): query is PersistedQuery =>
   !!query.extensions
 
-const parseString = (x: any) => (typeof x === 'string' ? JSON.parse(x) : x)
+const parseString = (x: unknown) => {
+  if (typeof x !== 'string') {
+    return x
+  }
+
+  try {
```

Comment:
I'm concerned with this try...catch masking any other errors. Wouldn't a simple ```if(x === 'undefined')``` be better in here ?

Reply:
sure! done in 3218578